firefox-ios/Client/FeatureFlags/LegacyFeatureFlagsManager.swift

```diff
@@ -100,9 +100,17 @@ class LegacyFeatureFlagsManager: HasNimbusFeatureFlags {
     }
 
     /// Set a feature that has a binary state to on or off
-    public func set(feature featureID: NimbusFeatureFlagID, to desiredState: Bool) {
+    public func set(feature featureID: NimbusFeatureFlagID, to desiredState: Bool, isDebug: Bool = false) {
         let feature = NimbusFlaggableFeature(withID: featureID, and: profile)
+        #if MOZ_CHANNEL_BETA || MOZ_CHANNEL_FENNEC
```

Comment:
given that we have the if statement, do we really need to flag on the channel? Are we just being careful? I can get behind that, but I'm curious for the rationale re: this

Reply:
mainly being careful, since the method itself is exposed outside of the debug setting. It was a bridge between my concern with coupling debug logic in the same method and adding the extra param to reuse the already existing method that we discussed during the call.

Reply:
Fair enough.

Reply:
thanks! I'm not strong on either position so happy to change if you feel that would be better for the team / codebase!